From a22a232d5b6bb301f0ca2b642d6a570981d2f7ea Mon Sep 17 00:00:00 2001
From: Alejandro Abdelnur <tucu@cloudera.com>
Date: Thu, 6 Sep 2012 12:12:24 -0700
Subject: [PATCH 0492/1357] HDFS-3895. hadoop-client must include commons-cli (tucu)

git-svn-id: https://svn.apache.org/repos/asf/hadoop/common/branches/branch-2@1381720 13f79535-47bb-0310-9956-ffa450edef68
(cherry picked from commit 87c6db314f02f1ba9c6848b088aa2154fdacdb3b)

Conflicts:

	hadoop-hdfs-project/hadoop-hdfs/CHANGES.txt
---
 hadoop-client/pom.xml                          |    4 ----
 hadoop-hdfs-project/hadoop-hdfs-httpfs/pom.xml |    4 ----
 2 files changed, 0 insertions(+), 8 deletions(-)

diff --git a/hadoop-client/pom.xml b/hadoop-client/pom.xml
index 25bc9c9..46628ad 100644
--- a/hadoop-client/pom.xml
+++ b/hadoop-client/pom.xml
@@ -40,10 +40,6 @@
       <scope>compile</scope>
       <exclusions>
         <exclusion>
-          <groupId>commons-cli</groupId>
-          <artifactId>commons-cli</artifactId>
-        </exclusion>
-        <exclusion>
           <groupId>commons-httpclient</groupId>
           <artifactId>commons-httpclient</artifactId>
         </exclusion>
diff --git a/hadoop-hdfs-project/hadoop-hdfs-httpfs/pom.xml b/hadoop-hdfs-project/hadoop-hdfs-httpfs/pom.xml
index d1a0940..d47e4fc 100644
--- a/hadoop-hdfs-project/hadoop-hdfs-httpfs/pom.xml
+++ b/hadoop-hdfs-project/hadoop-hdfs-httpfs/pom.xml
@@ -100,10 +100,6 @@
           <artifactId>stax-api</artifactId>
         </exclusion>
         <exclusion>
-          <groupId>commons-cli</groupId>
-          <artifactId>commons-cli</artifactId>
-        </exclusion>
-        <exclusion>
           <groupId>commons-httpclient</groupId>
           <artifactId>commons-httpclient</artifactId>
         </exclusion>
-- 
1.7.0.4

